Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -698,17 +701,24 @@ static GridArea insertIntoGrid(Grid& grid, RenderBox& child, const GridArea& are
     return clamped;
 }
 
-// TODO: Update name of functions to 'areMasonryRows'.
+// FIXME: Change isMasonryRows/Columns to areMasonryRows/Columns (249181)
```

Comment:
Nit: FIXME(249181): ...

Reply:
Fixed

Reply:
I threw up a quick pr to fix this https://github.com/WebKit/WebKit/pull/7625.

This really does not fit this patch description.

Reply:
You might need to rebase this patch as the name change landed.